pkg/controllers/reallocation/utilization.go

```diff
@@ -32,7 +32,7 @@ type Utilization struct {
 	kubeClient client.Client
 }
 
-func (u *Utilization) Reconcile(ctx context.Context, provisioner *v1alpha1.Provisioner) error {
+func (u *Utilization) Reconcile(ctx context.Context, provisioner *v1alpha2.Provisioner) error {
```

Comment:
Don't you need to add a check here to not do any work if `ttlSecondsAfterUnderutilization` is not set?

Reply:
This check is in controller.go